app/recordtransfer/views/post_submission.py

```diff
@@ -153,40 +139,44 @@ def get_form_kwargs(self) -> dict[str, Any]:
 
     def form_valid(self, form: BaseModelForm) -> HttpResponse:
         """Handle valid form submission."""
-        response = super().form_valid(form)
-        messages.success(self.request, self.success_message)
-        return response
+        super().form_valid(form)
+        response = render(
+            self.request,
+            self.template_name,
+            self.get_context_data(),
+        )
+        return trigger_client_event(
+            response,
+            "showSuccess",
+            {
+                "value": self.update_success_message,
+            },
+        )
 
     def form_invalid(self, form: BaseModelForm) -> HttpResponse:
         """Handle invalid form submission."""
-        messages.error(
-            self.request,
-            self.error_message,
+        response = super().form_invalid(form)
+        return trigger_client_event(
+            response,
+            "showError",
+            {
+                "value": self.update_error_message,
```

Comment:
Same comment as above:

```suggestion
                "value": gettext("There was an error updating the group"),
```